src/llmcompressor/modifiers/quantization/gptq/base.py

```diff
@@ -240,12 +279,10 @@ def initialize_compression(
             args = self._pruning_arguments()
             comp_cls = self._compression_class()
             compressor = LayerCompressor(comp_cls, self.model, layer, idx, name, args)
-            if not self.sequential_update:
-                # add all batch processing hooks before the forward pass
-                compressor.pre_compress()
+            compressor.pre_compress()
```

Comment:
I don't think we want to change this. `pre_compress` triggers the initialization of the `GPTQWrapper`s, which allocates the memory for the Hessians. We lose the memory benefit of sequential updates by allocating the Hessians up front

Reply:
Makes sense, I was wondering why we were delaying the precompression